### DISCUSSION TITLE
在AX86U上刷梅林装ALIST插件，挂载阿里云但过一段时间就失效

### DISCUSSION THREAD

**Discussion by X-Will-X** [AlistGo/alist#3373] [Q&A]
参考https://alist.nn.ci/zh/guide/drivers/aliyundrive.html 教程操作获取了刷新令牌后
可以添加成功了，也可以加载出来当下的云盘内容；
添加成功后云盘里新添加内容会刷新不出来，然后过一段时间就会提示
Post "https://auth.aliyundrive.com/v2/account/token": dial tcp [2408:4000:200::1c8]:443: connect: cannot assign requested address

（我看每次获取的刷新令牌貌似都是变化的？ 试了下手机上stream抓取的也一样，当下可以添加成功，后续刷新就不行了）
（补充：Alist插件程序版本：v3.10.1 面板版本：3.10.1）

**Comment by anwen-anyi** [comment]
这是属于你的网络问题吧~ DNS 还是啥总之是类似于网络波动导致的 在TG也见到过类似和你一样的
